### PR TITLE
[UT] Add ut for test predicate not change itself

### PR DESCRIPTION
**Pull request by Youngwb**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Add UT for #5970

### CODE REVIEW

**Comment by Youngwb** [comment]
run starrocks_fe_unittest

File: fe/fe-core/src/test/java/com/starrocks/sql/plan/DistributedEnvPlanWithCostTest.java [hunk 1]
```diff
@@ -1184,10 +1184,63 @@ public void testStringMinMaxPredicate() throws Exception {
...
         sql = "select Min(C_NAME) as min, Max(C_NAME) as max from customer having max < '1234'";
         costPlan = getCostExplain(sql);
-        System.out.println(costPlan);
         assertContains(costPlan, "* min-->[-Infinity, Infinity, 0.0, 25.0, 1.0] ESTIMATE\n" +
                 "  |  * max-->[-Infinity, Infinity, 0.0, 25.0, 1.0] ESTIMATE");
         assertContains(costPlan, "having: [11: max, VARCHAR, true] < '1234'\n" +
                 "  |  cardinality: 1");
     }
+
+    // https://github.com/StarRocks/starrocks/issues/5970
```
Comment (Seaven) [inline fe/fe-core/src/test/java/com/starrocks/sql/plan/DistributedEnvPlanWithCostTest.java:1193]:
```suggestion
```

File: fe/fe-core/src/test/java/com/starrocks/sql/plan/DistributedEnvPlanWithCostTest.java [hunk 2]
```diff
@@ -1184,10 +1184,63 @@ public void testStringMinMaxPredicate() throws Exception {
...
+    @Test
+    public void testSQLSmithGenQuery() throws Exception {
```
Comment (Seaven) [inline fe/fe-core/src/test/java/com/starrocks/sql/plan/DistributedEnvPlanWithCostTest.java:1195]: give a simple sql in ut

**Review by kangkaisen — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by Seaven — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Pull request merged**